storage/src/storage/ArrayStorage.java

```diff
@@ -0,0 +1,109 @@
+package storage;
+
+import model.Resume;
+import storage.exception.FullStorageException;
+import storage.exception.ResumeAlreadyStoredException;
+import storage.exception.ResumeNotFoundException;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public abstract class ArrayStorage implements Storage {
+    protected static final int NOT_EXISTING_INDEX = -1;
+    protected static final int DEFAULT_MAX_STORAGE_SIZE = 100000;
+    protected int maxSize;
+    protected int size;
+
+    protected Resume[] storage;
+
+    public ArrayStorage() {
+        maxSize = DEFAULT_MAX_STORAGE_SIZE;
+    }
+
+    protected ArrayStorage(int size) {
+        this.storage = new Resume[size];
+        this.maxSize = size;
```

Comment:
It's better to avoid variable shadowing. You already have `size`

Reply:
Done